platform/tron/api.go

```diff
@@ -25,17 +25,24 @@ func (p *Platform) Coin() coin.Coin {
 }
 
 func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
-	srcTxs, err := p.client.GetTxsOfAddress(address, "")
+	Txs, err := p.client.GetTxsOfAddress(address, "")
 	if err != nil {
 		return nil, err
 	}
 
+	if len(Txs) == 0 {
```

Comment:
why not do `if err != nil && len(Txs) == 0` ?

Reply:
https://github.com/trustwallet/blockatlas/pull/357/commits/f47a6982c4bc5af56d8b5fe366caf2e55eff9c15